pkg/migration/runner.go

```diff
@@ -407,6 +420,17 @@ func (r *Runner) setup(ctx context.Context) error {
 		if err := r.createCheckpointTable(ctx); err != nil {
 			return err
 		}
+
+		// The sentinelTable is "registered" separately from creating it.
+		// This helps to support the case where a user manually creates
+		// the sentinel table after a migration is started.
+		r.registerSentinelTable()
```

Comment:
Because of the previous suggestion about not in the struct, we can remove register.
```suggestion
		// The sentinelTable is is created if the user wants to defer cutover.
		// If they don't want to, we don't need to create it.
```